### ISSUE TITLE
pm grant command syntax wrong?

### ISSUE THREAD

**Issue by mdonz** [MuntashirAkon/CaptivePortalController#9]
Hi

thank you for your work on this app which I would like to use (avoid Google's default captiveportal). 

This is on a Xiaomi M11 (model: M2101K9G) with current MIUI 14.0.6. When running adb "pm grant" command. Unsure what I am doing wrong? Has the command syntax changed? Thanks for any advice.

sudo ./adb devices
[sudo] password for michael: 
List of devices attached
cb566fc4	device  

pm grant io.github.muntashirakon.captiveportalcontroller android.permission.WRITE_SECURE_SETTINGS
Usage: pm [action] [targets]
-1,--on targets      Power on targets
-0,--off targets     Power off targets
-c,--cycle targets   Power cycle targets
-q,--query-all       Query power state of all targets
-Q,--query targets   Query power state of specific targets

**Comment by MuntashirAkon** [comment]
You're using your distro's `pm` (power management) command. Run it this way:

```sh
adb shell pm grant io.github.muntashirakon.captiveportalcontroller android.permission.WRITE_SECURE_SETTINGS
```

**Comment by mdonz** [comment]
A, thanks for that.
I did as advised and got one step further. It's now stating:

sudo ./adb shell pm grant io.github.muntashirakon.captiveportalcontroller android.permission.WRITE_SECURE_SETTINGS

Exception occurred while executing 'grant':
java.lang.SecurityException: grantRuntimePermission: Neither user 2000 nor current process has android.permission.GRANT_RUNTIME_PERMISSIONS.

I searched about that problem and found many areas where users are exchanging recommendations what to enable or disable under developer options. It seems in my scenario (MIUI) there are more changes needed than for others. I believe I tried them all but to no avail. I suspect to do something wrong or to misunderstand but have given up for now.

Thanks again for the fast help.
